lib/buttons_tabbar.dart

```diff
@@ -326,16 +326,34 @@ class _ButtonsTabBarState extends State<ButtonsTabBar>
         ),
         animationValue);
 
+    var margin = EdgeInsets.only(
+      top: widget.buttonMargin.top,
+      bottom: widget.buttonMargin.bottom,
+      left: widget.buttonMargin.left / 2,
+      right: widget.buttonMargin.right / 2,
+    );
+
```

Comment:
```suggestion
    EdgeInsets margin;
```

Reply:
Implemented in 87ad9ff4ecbcdd91d66b82d586bee49de89568e0